pkg/utils/arguments.go

```diff
@@ -0,0 +1,29 @@
+package utils
+
+import (
+	"os"
+
+	"gopkg.in/yaml.v3"
+)
+
+func HandleRepositoryValues(repos []string, filePath string) (map[string]bool, error) {
+	uniqueRepoURLs := make(map[string]bool)
```

Comment:
I think a `map[string]struct{}` may be slightly more efficient since it essentially creates an empty member on the right side which effectively implements a "set". But that's more a suggestion: take it or leave it!